app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,45 @@
+class FriendshipsController < ApplicationController
+  before_action :authenticate_user!
+  def create
+    @friendship = current_user.friendships.build(friends_params)
+    if friends_params[:user].id != friends_params[:friend_id] && @friendship.save
+      flash[:notice] = "Friend request sent!"
+      redirect_to :back
+    else
+      flash[:notice] = "Unable to send friend request."
+      redirect_to :back
+    end
+  end
+
+  def update
```

Comment:
I'd make this a named action like accept_request. Hijacking update for this is a little hack-y.